std_msgs/Header header
time pose_ts
float64 x
float64 y
float64 teta
float64 x_raw
float64 y_raw
float64 teta_raw
